src/hooks/useApiInstance.ts

```
@@ -0,0 +1,69 @@
+
+  // simple api wrapper for easy access of the http methods
+  return {
+    get<TData>(
+      url: string,
+      options: Omit<FetchOptions<"json">, "method" | "body"> = {}
+    ) {
+      return fetch<TData>(url, options);
+    },
+    put<TData = unknown>(
+      url: string,
+      body: Record<string, any>,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)